.storybook/preview.tsx

```diff
@@ -11,7 +11,6 @@ const preview: Preview = {
 		backgrounds: {
 			default: 'light',
 		},
-		actions: { argTypesRegex: '^on[A-Z].*' },
```

Comment:
Everything else looks great, was wondering why we are removing this line? I didn't see anything in the v8 docs about this keyword being deprecated so just wanted to check!

Reply:
Good eye!

That line was boilerplate from the initial setup, and is for a feature we aren't using ([Actions](https://storybook.js.org/docs/essentials/actions)). The v8 migration script flagged it for suggested removal, since it's no longer the [recommended](https://storybook.js.org/docs/essentials/actions#via-storybooktest-fn-spy-function) way of mocking component methods. It's still _supported_, but not recommended:

> However, [[automatically matching args](https://storybook.js.org/docs/essentials/actions#automatically-matching-args)] is not the recommended way of writing actions. That's because automatically inferred args are not available as spies in your play function. If you use argTypesRegex and your stories have play functions, you will need to also define args with the fn utility to test them in your play function.

Reply:
Roger that! Sounds like it's approval time 😎